contributor-docs/adrs/adr-021-component-adoption.md

```diff
@@ -0,0 +1,54 @@
+# Component adoption
```

Comment:
Should we consider moving this ADR into our internal Primer repo? The content seems a bit specific to how we propose component adoption within consuming apps at GitHub, particularly with references to strategic services, the accessibility scorecard, etc. Or do we think contributors at large will benefit from this content?

Reply:
This is a great question! I wasn't sure if this content is a good candidate for an ADR at first and also since we added the same content to our internal private repo, I think we can close this PR 🙌🏻

let me know though if you have any concern!

Reply:
No concerns, as long as we make a plan to move this content into our internal team docs as I think it is still very relevant there!

Reply:
Already merged it to our internal repo ([PR link reference - GitHub stuff only](https://github.com/github/primer/pull/3725)) 😊